packages/account/src/providers/transaction-summary/operations.ts

```diff
@@ -111,31 +114,34 @@ export function getReceiptsMessageOut(receipts: TransactionResultReceipt[]) {
 }
 
 /** @hidden */
-const mergeAssets = (op1: Operation, op2: Operation) => {
+type Asset = {
+  assetId: string;
+  amount: BNInput;
+};
+
+export function mergeAssets(op1: Operation, op2: Operation): Asset[] {
   const assets1 = op1.assetsSent || [];
```

Comment:
Can `assetsSent` contain multiple of the same assets in a single operation? I assume not based upon the the [`getAssetsSent`](`https://github.com/FuelLabs/fuels-ts/pull/2839/files#diff-622c34b0c74e2ef212c2a5002c5b0ccc57154c72ca57644a3a57bdbe98594d01R267-R276) implementation.

## Example

```jsonc
{
  type: OperationName.transfer,
  assetsSent: [
    {
      amount: 100,
      assetId: '0x0000000000000000000000000000000000000000000000000000000000000000',
    },
    {
      amount: 100,
      assetId: '0x0000000000000000000000000000000000000000000000000000000000000000',
    },
  ]
  ...
}

Reply:
That's a good point, I can't seem to find an instance where `assetsSent` is ever more than one operation coin, so I am not sure why [the type](https://github.com/FuelLabs/fuels-ts/blob/mc/refactor-get-operations-refactor/packages/account/src/providers/transaction-summary/types.ts#L90) is of `Array<OperationCoin>;`

@Torres-ssf it seems like more than one entry is a [bug](https://github.com/FuelLabs/fuels-ts/pull/1400#issuecomment-1952541407) based on previous conversations. Should the `assetsSent` property be refactored to just an `OperationCoin` ?

Reply:
@petertonysmith94 @maschad From my understanding the `assetsSend` will not contain multiple entries of the same Asset ID in a single operation. However, it can include multiple entries of different asset IDs transferred from one account address to another. CC @LuizAsFight

@maschad On the mentioned issue, the bug was something else. In this case, there was one additional entry within `assetsSent` that shouldn't exist, because only one asset was being transferred.

Reply:
@Torres-ssf thanks for the knowledge - I'll resolve this one.